LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -155,6 +174,92 @@
+
++NewBehaviors=(BehaviorName=TargetHitChanceNotHigh, NodeType=Inverter, Child[0]=TargetHitChanceHigh)
++NewBehaviors=(BehaviorName=AttemptOnlyIfChanceIsHigh, NodeType=Sequence, Child[0]=TargetHitChanceNotHigh, Child[1]=AddToTargetScore_-1000)
++NewBehaviors=(BehaviorName=SSAttemptOnlyIfChanceIsHigh, NodeType=Successor, Child[0]=AttemptOnlyIfChanceIsHigh)
+
+;Technically avatars also use this tree, but their psi is so high that the change does literally nothing for them
++BehaviorRemovals="PsiEvaluateTargetsMindControl"
++NewBehaviors=(BehaviorName=PsiEvaluateTargetsMindControl, NodeType=Sequence, Child[0]=SetNextTarget, Child[1]=TargetScoreMindControl, Child[2]=TargetScoreHighestSoldierRank, Child[3]=TargetScoreHealth_HighFirst, Child[4]=TargetScoreInvalidateCivilians, Child[5]=AvoidBoundAndPanickedTargets, Child[6]=SSAttemptOnlyIfChanceIsHigh, Child[7]=UpdateBestTarget)
+
+; ----------------------------------------------------------------------------------------------------------
+; --------------------------- ADVENT SPECTRE --------------------------------------------------
+; ----------------------------------------------------------------------------------------------------------
+
+
++NewBehaviors=(BehaviorName=ShadowbindHPThreshold, NodeType=StatCondition, Param[0]=eStat_HP, Param[1]=">=", Param[2]="50%")
++NewBehaviors=(BehaviorName=ShadowbindAllyCheck, NodeType=StatCondition, Param[0]=VisibleAllyCount, Param[1]=">=", Param[2]=2)
+
++BehaviorRemovals="TryShadowbind"
++NewBehaviors=(BehaviorName=TryShadowbind, NodeType=Sequence, Child[0]=IsAbilityAvailable-Shadowbind, Child[1]=SetTargetStack-Shadowbind, Child[2]=SelectNLREnemyTargetRandomly, Child[3]=HasValidTarget-Shadowbind, Child[4]=SelectRevealedShadowbindOption)
+
++BehaviorRemovals="TryShadowbindIfNotLastAction"
++NewBehaviors=(BehaviorName=TryShadowbindIfNotLastAction, NodeType=Sequence, Child[0]=NotLastActionPoint, Child[1]=ShadowbindHPThreshold, Child[2]=ShadowbindAllyCheck, Child[3]=TryShadowbind)
+
++BehaviorRemovals="SpectreStealthSelector"
++NewBehaviors=(BehaviorName=SpectreStealthSelector, NodeType=Selector, \\
+	Child[0]=AdvanceCoverIfNoEnemiesVisible, \\
+	Child[1]=PostVanishMovement, \\
+	Child[2]=TryHighPriorityShot, \\
+	Child[3]=TryShadowbindIfNotActive, \\
+	Child[4]=TryMoveThenHorror, \\
+	Child[5]=AdvanceCoverMove, \\
+	Child[6]=DoIfFlankedMove, \\
+	Child[7]=TryShootOrReloadOrOverwatch, \\
+	Child[8]=HuntEnemyWithCover)
+
++BehaviorRemovals="SpectreStealthSelector"
++NewBehaviorsBehaviors=(BehaviorName=SpectreStealthSelector, NodeType=Selector, \\
+			Child[0]=PostShadowbindMove, \\
+			Child[1]=TryHighPriorityShot, \\
+			Child[2]=TryShadowbindIfNotActive, \\
+			Child[3]=TryVanish, \\
+			Child[4]=TryMoveThenHorror, \\
+			Child[5]=TryShootOrReloadOrOverwatch, \\
+			Child[6]=HuntEnemyWithCover)
+
```

Comment:
Duplicate entries?

Reply:
uh, no, but it still is a mistake, the latter is supposed to be VisibleSpectreBranch.